criu/kerndat.c

```diff
@@ -1133,6 +1136,36 @@ static int kerndat_has_pidfd_getfd(void)
 	return ret;
 }
 
+static int kerndat_has_nftables_concat(void)
+{
+#if defined(CONFIG_HAS_NFTABLES_LIB_API_0) || defined(CONFIG_HAS_NFTABLES_LIB_API_1)
+	struct nft_ctx *nft;
+	int ret = 0;
+
+	nft = nft_ctx_new(NFT_CTX_DEFAULT);
+	if (!nft)
+		return -1;
+
+	kdat.has_nftables_concat = true;
+	if (NFT_RUN_CMD(nft, "add table inet CRIU")) {
+		ret = -1;
+		goto nft_ctx_free_out;
```

Comment:
When `NFT_RUN_CMD()` fails here, `kdat.has_nftables_concat` would remain set to `true`.